app/models/contestant.rb

```diff
@@ -1,3 +1,8 @@
 class Contestant < ApplicationRecord
+  has_many :contestant_projects
+  has_many :projects, through: :contestant_projects
 
+  def find_contestant_projects
+    Project.joins(:contestants).where("contestants.id = #{self.id}")
```

Comment:
This query is correct but there's an even easier solution: `projects.pluck(:name)`

Reply:
Since we have the association set up, we can call projects directly on an instance of a contestant